player/command.c

```diff
@@ -2004,6 +2004,14 @@ static int get_track_entry(int item, int action, void *arg, void *ctx)
     struct replaygain_data rg = has_rg ? *track->stream->codec->replaygain_data
                                        : (struct replaygain_data){0};
 
+    struct mp_tags *tags = track->stream ? track->stream->tags : (struct mp_tags *){0};
+    char **tag_list = talloc_zero_array(NULL, char *, tags->num_keys * 2 + 1);
+    for (int i = 0; i < tags->num_keys; i++) {
+        tag_list[2 * i] = talloc_strdup(tag_list, tags->keys[i]);
+        tag_list[2 * i + 1] = talloc_strdup(tag_list, tags->values[i]);
+    }
+    tag_list[tags->num_keys * 2] = 0;
```

Comment:
Not needed, if you do `talloc_zero_array`, though, you may do `talloc_array` and keep this.

Reply:
It segfaults without this.

Reply:
Okay I lied. I swear it was overflowing earlier but it seems like it's not anymore.